File: innovation_app/lib/models/plan.dart
class Plan {
  final int id;
  final String name;
  final double price;

  Plan({
    required this.id,
    required this.name,
    required this.price,
  });

  factory Plan.fromJson(Map<String, dynamic> json) {
    return Plan(
      id: json['id'],
      name: json['name'],
      price: json['price'].toDouble(),
    );
  }
}

File: innovation/innovation/app/models/plan.py
from sqlalchemy import Column, Integer, String, JSON
from app.db.base import Base

class Plan(Base):
    __tablename__ = "plans"

    id = Column(Integer, primary_key=True)
    name = Column(String, unique=True)
    price = Column(Integer)
    features = Column(JSON)
    # ex: ["pdf", "history", "ia"]
